packages/beacon-node/test/spec/presets/ssz_static.test.ts

```diff
@@ -46,8 +46,8 @@ const sszStatic =
     const sszType =
       // Since lightclient types are not updated/declared at all forks, this allForksLightClient
       // will help us get the right type for lightclient objects
-      ((ssz.allForksLightClient[fork as ForkLightClient] || {}) as Types)[typeName] ||
-      (ssz[fork] as Types)[typeName] ||
+      ((sszTypesFor(fork as ForkLightClient) || {}) as Types)[typeName] ||
+      (sszTypesFor(fork) as Types)[typeName] ||
```

Comment:
those two lines are essentially the same, we can dedupe. We might also wanna update the comment in L47-48